CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -0,0 +1,88 @@
+# Changelog
+
+## [Unreleased](https://github.com/diem/diem/tree/HEAD)
+
+[Full Changelog](https://github.com/diem/diem/compare/diem-core-v1.2.0...HEAD)
+
+**Fixed bugs:**
+
+- \[Bug\] Got BadProxy on release 1.2.0 when running cli [\#8297](https://github.com/diem/diem/issues/8297)
```

Comment:
Did you see my comment on the internal doc? It sounds like at least some of these were due to configuration changes and should not really be listed as "fixed" bugs.

Reply:
Dang it, i didn't see it (the contrast of the highlight is nearly indistinguishable to my eyes) until you pointed out here.   

I agree.  These aren't really bug fixes as there is no PR attached. I will remove them.